test/nitro/transfer-sol.test.ts

```diff
@@ -52,7 +52,7 @@ describe("transfer (solidity)", function () {
       exitRequest
     );
 
-    expect(gasEstimate.toNumber()).to.equal(45780);
+    expect(gasEstimate.toNumber()).to.equal(45876);
```

Comment:
Following up from discussion last week - I am really enjoying this pattern. Seeing the `96` gas cost of that conditional is very instructive.

Reply:
Yeah it's nice. I'm still a bit vexed by other changes to gas consumption that I am seeing, however. I've spotted that simply adding a test case "after" an existing test case (with no change to the solidity code) can change the gas consumption of that existing case. 

My rough conjecture is that the order of the test cases is not predictable (because they are run async), and the later test may run before the earlier one. So that would explain away the 'backwards in time" causation...

And then perhaps there  is some differing cost now because the account submitting the transaction has a higher nonce? Not quite sure if that makes sense.

When I get time I will try to reproduce this scenario.